src/js/edu-benefits/5490/config/form.js

```diff
@@ -405,24 +404,28 @@ const formConfig = {
               status: {
                 'ui:title': 'What\'s your current high school status?',
                 'ui:options': {
-                  labels: highSchoolStatusLabels
+                  labels: highSchoolStatusLabels,
+                  expandUnderClassNames: 'schemaform-expandUnder-indent'
                 }
               },
-              highSchoolOrGedCompletionDate: _.assign(
+              'view:highSchoolOrGedCompletionDate': _.assign(
                 date.uiSchema('When did you earn your high school diploma?'), {
                   'ui:options': {
                     hideIf: form => {
                       const status = _.get('highSchool.status', form);
-                      return status !== 'graduated' && status !== 'ged';
-                    }
+                      return status !== 'graduated';
```

Comment:
Maybe add a TODO here that this will need to change to `status !== 'graduated' && status != 'discontinued'` once the BE schema change is done.